dist/index.js

```diff
@@ -3702,6 +3702,7 @@ function asPromise(normalizedOptions) {
                     request._beforeError(new types_1.HTTPError(response));
                     return;
                 }
+                request.destroy();
```

Comment:
Please revert the changes to `dist/index.js`. We do not commit directly to this file except when the team is doing a release.

Reply:
Ah, my apologies, I didn't realize! I pushed a commit removing this particular change